Nightly Build: PC fails to compile on debug with a OpenImageIO error

**Describe the bug**
PC is failing to compile on debug on nightly build due the following an error with OpenImageIO while creating Atom_RPI.Builders library:

````
[2022-06-21T05:45:36.552Z] D:\workspace\3rdParty\packages\openimageio-opencolorio-2.3.12.0-rev2-windows\OpenImageIO\include\OpenImageIO/fmath.h(2384,5): error C4777: 'fprintf' : format string '%zd' requires an argument of type 'unsigned __int64', but variadic argument 5 has type 'OpenImageIO_v2_3::span_strided<const float,-1>::size_type' [D:\workspace\o3de\build\windows\External\GradientSignal-2eb7fe95\Code\GradientSignal.Editor.Static.vcxproj]
[2022-06-21T05:45:36.553Z]       DASSERT_MSG (y.size() >= 2, "interpolate_linear needs at least 2 knot values (%zd)", y.size());
[2022-06-21T05:45:36.553Z]       ^
[2022-06-21T05:45:36.553Z]   D:\workspace\3rdParty\packages\openimageio-opencolorio-2.3.12.0-rev2-windows\OpenImageIO\include\OpenImageIO/fmath.h(2384,5): note: the sizes of types 'OpenImageIO_v2_3::span_strided<const float,-1>::size_type' and 'unsigned __int64' might differ on other platforms
[2022-06-21T05:45:36.553Z]       DASSERT_MSG (y.size() >= 2, "interpolate_linear needs at least 2 knot values (%zd)", y.size());
[2022-06-21T05:45:36.553Z]       ^
[2022-06-21T05:45:36.553Z]   D:\workspace\3rdParty\packages\openimageio-opencolorio-2.3.12.0-rev2-windows\OpenImageIO\include\OpenImageIO/fmath.h(2384,5): note: consider using '%td' in the format string
[2022-06-21T05:45:36.553Z]       DASSERT_MSG (y.size() >= 2, "interpolate_linear needs at least 2 knot values (%zd)", y.size());
[2022-06-21T05:45:36.553Z]       ^
[2022-06-21T05:45:37.117Z]      Creating library D:/workspace/o3de/build/windows/lib/debug/Atom_RPI.Builders.lib and object D:/workspace/o3de/build/windows/lib/debug/Atom_RPI.Builders.exp
````

**Failed Jenkins Job Information:**
https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE_periodic-incremental-daily/detail/development/15/pipeline/1087/

**Attachments**
[log.txt](https://github.com/o3de/o3de/files/8950560/log.txt)

--- a/Gems/GradientSignal/Code/Source/Editor/EditorGradientBakerComponent.cpp
+++ b/Gems/GradientSignal/Code/Source/Editor/EditorGradientBakerComponent.cpp
@@ -14,7 +14,9 @@
 #include <GradientSignal/Ebuses/GradientPreviewRequestBus.h>
 #include <GradientSignal/Ebuses/ImageGradientRequestBus.h>
 
+AZ_PUSH_DISABLE_WARNING(4777, "-Wunknown-warning-option")
 #include <OpenImageIO/imageio.h>
+AZ_POP_DISABLE_WARNING
 
 namespace GradientSignal
 {

